runtime/testy/runtime.py

```diff
@@ -267,6 +295,64 @@ def startHibike(badThingsQueue, stateQueue, pipe):
   except Exception as e:
     badThingsQueue.put(BadThing(sys.exc_info(), str(e)))
 
+def ensure_is_function(tag, val):
+  if inspect.iscoroutinefunction(val):
+    raise RuntimeError("{} is defined with `async def` instead of `def`".format(tag))
+  if not inspect.isfunction(val):
+    raise RuntimeError("{} is not a function".format(tag))
+
+def ensure_not_overridden(module, name):
+  if hasattr(module, name):
+    raise RuntimeError("Student code overrides `{}`, which is part of the API".format(name))
+
+def clarify_coroutine_warnings(exception_cell):
+    """
+    Python's default error checking will print warnings of the form:
+        RuntimeWarning: coroutine '???' was never awaited
+
+    This function will inject an additional clarification message about what
```

Comment:
This should be changed to say:

This function will upgrade such a warning to a fatal error, while also injecting a additional clarification message about possible causes.